src/interop/lionweb.ts

```diff
@@ -29,7 +30,7 @@ import {STARLASU_LANGUAGE} from "./lionweb-starlasu-language";
 export {STARLASU_LANGUAGE} from "./lionweb-starlasu-language";
 
 export const ASTNode = STARLASU_LANGUAGE.entities.find(e => e.name == "ASTNode")! as Concept;
-export const PositionFeature = ASTNode.features.find(f => f.name == "position")! as Containment;
+export const PositionFeature = ASTNode.features.find(f => f.name == "position")! as Property;
 export const PositionClassifier = PositionFeature.type!;
```

Comment:
I would not call those Classifiers as Classifier is a Concept, an Interface or an Annotation. This is a PrimitiveType

Reply:
Right, I'm changing it